conf/scopes.json

```diff
@@ -959,5 +959,15 @@
     "key":"read:ppns-boxes",
     "name":"PPNS API",
     "description":"Allow read access of PPNS boxes"
+  },
+  {
+    "key":"read:self-assessment-assist",
+    "name":"Read self assessment assist data",
+    "description":"Allow ability to read the self assessment data as a report"
+  },
+  {
+    "key":"write:self-assessment-assist",
+    "name":"Write self assessment assist data",
+    "description":"Allow ability to write acknowledgement"
```

Comment:
As this is the message shown to an end user granting permission, is it possible to add more info (eg acknowledgement of what)?

Reply:
As discussed happy to merge as is - please do update once you have feedback from your technical writer.